101-hdinsight-secure-vnet/azuredeploy.json

```
@@ -47,7 +47,9 @@
       "name": "[concat(parameters('clusterName'),'-vnet')]",
       "addressSpacePrefix": "10.0.0.0/16",
       "subnetName": "subnet1",
-      "subnetPrefix": "10.0.0.0/24"
+      "subnetPrefix": "10.0.0.0/24",
+      "id": "[resourceId('Microsoft.Network/virtualNetworks', concat(parameters('clusterName'),'-vnet'))]",
+      "subnet": "[concat(resourceId('Microsoft.Network/virtualNetworks', concat(parameters('clusterName'),'-vnet')), '/subnets/subnet1')]"
```

```suggestion
      "subnet": "[resourceId('Microsoft.Network/virtualNetworks/subnets', concat(parameters('clusterName'),'-vnet')), 'subnet1')]"
```

And then remove the id above